site/styles/_availability-tags.scss

```diff
@@ -71,3 +71,25 @@ s72-plan-label {
   font-size: 12px;
   text-align: center;
 }
+
+s72-live-lozenge {
+  .live-lozenge {
+    background-color: var(--live-lozenge-background-color);
+    border-radius: 100px;
+    font-size: 12px;
+    line-height: 30px;
```

Comment:
The circle isn't quite aligning vertically with the text, its appears a pixel or two below on my screen 
![image](https://user-images.githubusercontent.com/32156362/187806137-260a090b-eb12-4fea-b21a-de9922723571.png)

Reply:
I've gone with a negative margin hack to bump the dot a bit higher.  I think this has something to do with our icon glyphs not fitting into their EM squares as expected.  Although... looking at what is in that glyph... I would expect it to appear too high, not too low.

Reply:
Not trying to be contentious but I feel like this is a better solution:
```
display: flex;
align-items: center;

Reply:
Definitely a better solution!